glide-core/src/protobuf/redis_request.proto

```diff
@@ -135,6 +135,7 @@ enum RequestType {
     Rename = 91;
     DBSize = 92;
     Brpop = 93;
+    Blpop = 100;
```

Comment:
Why doesn't `blpop` have the consecutive number?

Reply:
There is a number of PRs are pending with numbers in that range